lib/galaxy_test/selenium/test_uploads.py

```diff
@@ -173,6 +178,8 @@ def test_rules_example_2_list(self):
         # self.screenshot("rules_example_2_5_download_complete")
 
     @selenium_test
+    @pytest.mark.screenshot
```

Comment:
A lot of the tests generate screenshots. Maybe call it training_screenshot or tutorial_screenshot or something?

Reply:
I'd probably add another marker for training/tutorial if it makes sense to subset them.

Reply:
> I'd probably add another marker for training/tutorial if it makes sense to subset them.

I genuinely don't think it's something important, it's rather a minor detail. But I really need it for https://github.com/galaxyproject/ci-testing and if it helps to get it merged...